wolfcrypt/src/port/Espressif/esp_crt_bundle/cacrt_deprecated.pem

```diff
@@ -0,0 +1,198 @@
+##
```

Comment:
Why keep deprecated certs? Are these used?

Reply:
Good question. Yes, they are used. Not very intuitive without a sample app. Please [see this line of the Kconfig](https://github.com/gojimmypi/wolfssl/blob/63149c1b6510362884c69145c4c5af166c22dc96/IDE/Espressif/ESP-IDF/examples/esp_http_client_example/components/wolfssl/Kconfig#L276) that will be in the wolfSSL component, and copied here in the third option of the bundle config selection:

```
            config WOLFSSL_CUSTOM_CERTIFICATE_BUNDLE
                depends on WOLFSSL_CERTIFICATE_BUNDLE
                default n
                bool "Add custom certificates to the default bundle"
            config WOLFSSL_CUSTOM_CERTIFICATE_BUNDLE_PATH
                depends on WOLFSSL_CUSTOM_CERTIFICATE_BUNDLE
                string "Custom certificate bundle path"
                help
                    Name of the custom certificate directory or file. This path is evaluated
                    relative to the project root directory.

            config WOLFSSL_CERTIFICATE_BUNDLE_DEPRECATED_LIST
                bool "Add deprecated root certificates"
                depends on WOLFSSL_CERTIFICATE_BUNDLE && !WOLFSSL_CERTIFICATE_BUNDLE_DEFAULT_NONE
                help
                    Include the deprecated list of root certificates in the bundle.
                    This list gets updated when a certificate is removed from the Mozilla's
                    NSS root certificate store. This config can be enabled if you would like
                    to ensure that none of the certificates that were deployed in the product
                    are affected because of the update to bundle. In turn, enabling this
                    config keeps expired, retracted certificates in the bundle and it may
                    pose a security risk.

                    - Deprecated cert list may grow based based on sync with upstream bundle
                    - Deprecated certs would be be removed in ESP-IDF (next) major release
```

See also the mirrored functionality from the [mbedtls/Kconfig](https://github.com/espressif/esp-idf/blob/6673376297b921d438790d195014b860eaf8bb17/components/mbedtls/Kconfig#L350)